helpers/z_schema.js

```diff
@@ -2,7 +2,8 @@
 
 var ip = require('ip');
 /**
- * Uses JSON Schema validator z_schema to register custom formats.
+ * Uses JSON Schema validator z_schema to register custom formats. <br/>
```

Comment:
<br/> tag needed here?

Reply:
Wanted to show next line to a next line in generated docs. That was the only way I found to do it.

Reply:
Ok fine with me